src/theory/quantifiers/quantifiers_rewriter.cpp

```diff
@@ -987,9 +966,12 @@ bool QuantifiersRewriter::getVarElim(Node n,
   Kind nk = n.getKind();
   if (nk == NOT)
   {
-    return getVarElim(n[0], !pol, args, vars, subs);
+    n = n[0];
+    pol = !pol;
+    nk = n.getKind();
+    Assert(nk != NOT);
   }
-  else if ((nk == AND && pol) || (nk == OR && !pol))
+  if ((nk == AND && pol) || (nk == OR && !pol))
```

Comment:
`else if`?

Reply:
Actually no, I removed the recursion from the NOT case, so now it should fall through and trigger this.

Reply:
Ah, I did not notice that `nk` is updated.